changes/noissue.6.fix.rst

```diff
@@ -0,0 +1,4 @@
+Development: Fix issue with minimum-constraints-develop.txt that causing
+failure of make check_reqs. Added two pkgs to minimum-constraints-develop.txt:
+package Levenshtein  and python-Levenshtein used by safety and roman-numerals-py
+used by sphinx
```

Comment:
The "python-Levenshtein" package has been added by safety 3.3.0 and will be removed again in safety 3.3.1 because it is GPL-licensed. See https://github.com/pyupio/safety/issues/682. Since "safety" is the only package using "python-levenshtein" (or its renamed package "levenshtein"), I suggest fixing this issue by excluding safety 3.3.0 instead of adding any of the "levenshtein" packages.

Reply:
Done